sdk/web-pubsub/web-pubsub-client/samples/v1/typescript/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
samples under `samples/v1/typescript` and `samples/v1/javascript` are supposed to be published from the source under `samples-dev/` by using dev-tool, i.e., `npx dev-tool samples publish`.  Seems that this was hand-crafted?